lucene/analysis/kuromoji/src/tools/java/org/apache/lucene/analysis/ja/util/BinaryDictionaryWriter.java

```diff
@@ -35,9 +35,11 @@
 import org.apache.lucene.analysis.ja.dict.BinaryDictionary;
 
 public abstract class BinaryDictionaryWriter {
+  private final static int ID_LIMIT = 8192;
+
   protected final Class<? extends BinaryDictionary> implClazz;
   protected ByteBuffer buffer;
-  private int targetMapEndOffset = 0, lastWordId = -1, lastSourceId = -1;
+  protected int targetMapEndOffset = 0, lastWordId = -1, lastSourceId = -1;
```

Comment:
what happened here? why is this protected now?

Reply:
my bad - this was left this way from some earlier debugging efforts. I'll undo